### PR TITLE
frontend using ionic framework follow figma design

### PR DESCRIPTION
**Pull request by winnie-chaintope**

### CODE REVIEW

File: backend/actions/address_detail.js [hunk 1]
```diff
@@ -0,0 +1,45 @@
+const Client = require('bitcoin-core');
+const app = require('../app.js');
+
+const cl = new Client({
+  network: 'regtest',
+  username: 'user',
+  password: 'password',
+  port: 18443,
+});
+
```
Comment (azuchi) [inline backend/actions/address_detail.js:10]: Do not write credentials directly. If you want to run it locally, you can also specify the path of the tapyrus configuration file.

Comment (winnie-chaintope) [reply backend/actions/address_detail.js:10]: ok i will make the changes

Comment (azuchi) [reply backend/actions/address_detail.js:6]: In this implementation, the path is fixed, so it is environment-dependent. Therefore, it is better to read the path from the environment variable.

Comment (winnie-chaintope) [reply backend/actions/address_detail.js:7]: ok i changed to use the cross-env package

**Pull request merged**